src/pages/users/[id]/share.tsx

```diff
@@ -147,60 +161,48 @@ const SharePage = () => {
     setShareTelegram(event.target.checked);
   };
 
+  if (!profile) {
+    return <div>Loading...</div>;
+  }
+
   if (!user) {
     return <div>User not found</div>;
   }
 
   return (
```

Comment:
We need frontends for this image.png

Reply:
![image](https://github.com/nfc-denver/nfc-denver/assets/6290822/4631f3ff-5750-4638-a4b4-669b31f5a147)

Reply:
https://github.com/nfc-denver/nfc-denver/issues/29